tests/Query/UdpTransportExecutorTest.php

```diff
@@ -32,32 +32,32 @@ public function testCtorShouldAcceptNameserverAddresses($input, $expected)
 
     public static function provideDefaultPortProvider()
     {
-        return array(
-            array(
+        return [
+            [
                 '8.8.8.8',
                 'udp://8.8.8.8:53'
-            ),
-            array(
+            ],
+            [
                 '1.2.3.4:5',
                 'udp://1.2.3.4:5'
-            ),
-            array(
+            ],
+            [
                 'udp://1.2.3.4',
                 'udp://1.2.3.4:53'
-            ),
-            array(
+            ],
+            [
                 'udp://1.2.3.4:53',
                 'udp://1.2.3.4:53'
-            ),
-            array(
+            ],
+            [
                 '::1',
                 'udp://[::1]:53'
-            ),
-            array(
+            ],
+            [
                 '[::1]:53',
                 'udp://[::1]:53'
-            )
-        );
+            ]
+        ];
```

Comment:
Same as `ParserTest`, sadly GitHub won't let me suggest the changes due to deleted lines.

Reply:
Good catch, adjusted as suggested :+1: